modules/http/basic_auth_strong_password/__init__.py

```diff
@@ -37,7 +37,9 @@ def processor(self):
                 for data in data_dump:
                     data_json = json.loads(data)
                     ip = data_json["ip"]
-                    time_of_insertion = data_json["time"]
+                    date_log = data_json["time"]
+                    date = datetime.datetime.strptime(date_log[1:12],"%d/%b/%Y").strftime("%Y-%m-%d")
+                    time_of_insertion = date + ' ' + date_log[13:21]
```

Comment:
is there a lot of difference in date and time if we store from host instead of the client?

Reply:
the difference is in the formatting, in the description I put 2 different dates but that was not the objective.
[09 / Feb / 2021: 22: 51: 01 +0000] ---> 2021-02-09 22:51:01

Reply:
@dhirensr the difference is in the formatting, in the description I put 2 different dates but that was not the objective.
[09 / Feb / 2021: 22: 51: 01 +0000] ---> 2021-02-09 22:51:01

Reply:
oh i see now. I'm not sure how will it help, because before the API queries the date format is being fixed by this function (https://github.com/OWASP/Python-Honeypot/blob/1b8a738de37af13e373ba6d055300c8184a9df54/api/utility.py#L122).
@ChakshuGupta @Ali-Razmjoo : what do you think?

Reply:
@dhirensr I noticed this bug because the API could not retrieve the information relating to http credentials, fixed the bug everything works correctly.